### PR TITLE
[COST-5015] Retrieve azure disk capacities

### PR DESCRIPTION
**Pull request by myersCody**
## Jira Ticket

[COST-5015](https://issues.redhat.com/browse/COST-5015)

## Description

This change will create a table to house a mapping of product substrings related to Azure managed disks and their capacities in GiBs. 

## Testing

Hit masu endpoint, and check table:

```
DELETE 39
postgres=# select * from reporting_common_azurestoragecapacity;
 product_substring | capacity
-------------------+----------
 P1                |        4
 P2                |        8
 P3                |       16
 P4                |       32
 P6                |       64
 P10               |      128
 P15               |      256
 P20               |      512
 P30               |     1024
 P40               |     2048
 P50               |     4096
 P60               |     8192
 P70               |    16384
 P80               |    32767
 S4                |       32
 S6                |       64
 S10               |      128
 S15               |      256
 S20               |      512
 S30               |     1024
 S40               |     2048
 S50               |     4096
 S60               |     8192
 S70               |    16384
 S80               |    32767
 E1                |        4
 E2                |        8
 E3                |       16
 E4                |       32
 E6                |       64
 E10               |      128
 E15               |      256
 E20               |      512
 E30               |     1024
 E40               |     2048
 E50               |     4096
 E60               |     8192
 E70               |    16384
 E80               |    32767
(39 rows)
```

## Release Notes
- [ ] proposed release note

```markdown
* [COST-####](https://issues.redhat.com/browse/COST-####) Fix some things
```

### CODE REVIEW

**Review by maskarb — DISMISSED** [review]

File: koku/masu/celery/tasks.py [hunk 1]
```diff
@@ -381,6 +383,48 @@ def get_daily_currency_rates():
...
 
 
+@celery_app.task(name="masu.celery.scrape_azure_storage_capacities", queue=DEFAULT)
+def scrape_azure_storage_capacities():
+    """Task to retrieve the Azure disk capacities.
+
+    The Azure cost reports do not report disk capacities. Therefore, we retrieve
+    the disk capacities and product substring from their documentation repos.
+    """
+    # TODO: Should we move this url to an env var?
```
Comment (cgoodfred) [inline koku/masu/celery/tasks.py:393]: Shouldn't we resolve this TODO before we merge these changes?

**Review by cgoodfred — APPROVED** [review]

**Pull request merged**